edb/ir/scopetree.py

```diff
@@ -21,6 +21,8 @@
 
 from __future__ import annotations
 from typing import *
+if TYPE_CHECKING:
+    from typing_extensions import TypeGuard
```

Comment:
Don't forget to add a dependency to `setup.py`

Reply:
The stubs are mypy internal, and we aren't actually importing it yet at runtime